src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/Maintainability/AvoidUnusedResources.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using Analyzer.Utilities;
+using Analyzer.Utilities.Extensions;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Operations;
+
+namespace Microsoft.CodeQuality.Analyzers.Maintainability
+{
+    using static MicrosoftCodeQualityAnalyzersResources;
+
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+    public sealed class AvoidUnusedResources : DiagnosticAnalyzer
+    {
+        internal const string RuleId = "CA1514";
+        internal static readonly DiagnosticDescriptor Rule = DiagnosticDescriptorHelper.Create(
+            RuleId,
+            CreateLocalizableResourceString(nameof(AvoidUnusedResourcesTitle)),
+            CreateLocalizableResourceString(nameof(AvoidUnusedResourceMessage)),
+            DiagnosticCategory.Maintainability,
+            RuleLevel.Disabled,
```

Comment:
I am wondering if we should have this as `RuleLevel.Ide_Suggestion` instead of disabled by default. I understand that the rule will not report suggestion diagnostics by default as it is a compilation end analyzer, but having this as an Info diagnostic by default will mean that specifying any non-default value for [AnalysisMode](https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#analysismode) will bump this to a warning and start reporting unused resource warnings on command line builds. We can also implicitly pass in all .resx files in the project as additional files when `AnalysisMode` or `AnalysisLevel` properties end with `minimum`, `recommended` or `all` similar to the below added to MS.CA.Analyzers package: https://github.com/dotnet/roslyn-analyzers/blob/79cc00251332fe690d9cbba8f367c7b21a199bc0/src/Tools/GenerateDocumentationAndConfigFiles/Program.cs#L1601-L1608